network/game_test.go

```diff
@@ -0,0 +1,641 @@
+package network
+
+import (
+	"testing"
+
+	"github.com/joshprzybyszewski/cribbage/model"
+	"github.com/stretchr/testify/assert"
+)
+
+func modelCardsFromStrings(cs ...string) []model.Card {
+	hand := make([]model.Card, len(cs))
+	for i, c := range cs {
+		hand[i] = model.NewCardFromString(c)
+	}
+	return hand
+}
```

Comment:
This is likely going to be needed soon for executing game actions from the app; but I'm okay with it living here for now